rca/api_content/views.py

```diff
@@ -0,0 +1,13 @@
+from django.core.cache import cache
+from django.http import HttpResponseRedirect
+from django.views.generic import TemplateView
+
+
+class ApiContentCacheSettings(TemplateView):
+    template_name = "api_content/api_content_admin.html"
+
+
+def ApiContentCacheClear(request):
```

Comment:
Why use camel case for function-based views? https://docs.djangoproject.com/en/3.0/topics/http/views/ Django docs do not seem to use this. Is that what Wagtail does?

Reply:
no that's my error.... thanks